packages/app-desktop/gui/styles/inline-markdown.scss

```
@@ -0,0 +1,37 @@
+	p:last-child {
+		margin-bottom: 0;
+	}
+
+	ul, ol {
+		padding-left: var(--joplin-margin);
+	}
+
+	pre {
+		white-space: pre;
+		white-space: pre-wrap;
+		word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)